src/main/java/com/moksh/pastebin/controller/PasteBinController.java

```diff
@@ -0,0 +1,55 @@
+package com.moksh.pastebin.controller;
+
+import com.moksh.pastebin.entity.Paste;
+import com.moksh.pastebin.entity.request.PasteDTO;
+import com.moksh.pastebin.service.PasteBinService;
+import com.moksh.pastebin.service.implementation.PasteBinServiceImpl;
+import lombok.extern.slf4j.Slf4j;
+import static org.springframework.http.HttpStatus.CREATED;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
+
+@RestController
+@Slf4j
+public class PasteBinController {
+
+    private PasteBinServiceImpl pasteBinService;
+
+    @PostMapping("/")
+    @ResponseStatus(CREATED)
+    public ResponseEntity<Paste> createPaste(@RequestBody PasteDTO pasteDTO){
+
+        Paste paste = new Paste();
+        paste.setTtl(pasteDTO.getTtl());
```

Comment:
These DTO operations should be verified in controller, conversions in service layer.